packages/dds/tree/src/feature-libraries/sequence-field/moveEffectTable.ts

```diff
@@ -17,6 +17,7 @@ export type MoveEffectTable<T> = CrossFieldManager<MoveEffect<T>>;
  * Changes to be applied to a move mark.
  */
 export interface MoveEffect<T> {
+	// TODO: Does this need to be tagged?
```

Comment:
It probably doesn't anymore

Reply:
Scratch that. Composing already composite changesets could put us in a situation where we can't infer the revision tag here.

Reply:
If the change is from a composite changeset then the contents should already have tags internally. I don't think we should ever need the tag from this now.

Reply:
I was thinking about the case where the changes come from a composite changeset but that part of the changeset is from a single revision (meaning there wouldn't be any revision info internally). Do we correctly adopt the revision tag for that portion of the changeset as the new "ambient" revision on our way down?

Reply:
Evidently you were able to remove the tagging without negative impact so at the very least we don't need it for our current test coverage. I'm pretty sure I added a test when I added the tagging in this part of the code, so we must be good.